test/foundry/integration/the-merge-migration-single-token-vault/TheMergeMigrationSingleTokenBase.t.sol

```diff
@@ -222,9 +222,77 @@ abstract contract TheMergeMigrationSingleTokenBase is BaseTest {
       address(usdcTokenVault),
       true
     );
+
+    address[] memory list = new address[](1);
+    list[0] = FRANK;
+
+    _distributeUSDC(list, 1000e6);
+    // try stake/withdraw when fee rate is 0
+    vm.startPrank(FRANK);
+    uint256 balanceBeforeStake = USDC.balanceOf(FRANK);
+    USDC.approve(address(usdcTokenVault), 1000e6);
+
+    usdcTokenVault.stake(1000e6);
+    uint256 balanceAfterStake = USDC.balanceOf(FRANK);
+
+    assertEq(balanceBeforeStake - balanceAfterStake, 1000e6);
+    assertEq(usdcTokenVault.balanceOf(FRANK), 1000e6);
+
+    usdcTokenVault.withdraw(1000e6);
+
+    assertEq(usdcTokenVault.balanceOf(FRANK), 0);
+    assertEq(USDC.balanceOf(FRANK) - balanceAfterStake, 1000e6);
+
+    vm.stopPrank();
+
+    assertEq(USDC.balanceOf(FRANK), 1000e6);
+
     //  - Start a reward distribution process
     POWAAToken.transfer(address(usdcTokenVault), 6048000 ether); // 10 POWAA / sec
     usdcTokenVault.notifyRewardAmount(6048000 ether);
+
+    tokenVaultImpl.transferOwnership(FRANK);
+    assertEq(usdcTokenVault.getMasterContractOwner(), address(FRANK));
+
+    vm.prank(FRANK);
+    //  - Set Migration Option for usdcTokenVault
+    usdcTokenVault.setMigrationOption(
+      tokenVaultMigrator,
+      tokenVaultReserveMigrator,
+      THE_MERGE_BLOCK + 1000,
+      address(linearFeeModel),
+      USDC_ETH_V3_FEE,
+      WITHDRAWAL_TREASURY,
+      WITHDRAWAL_TREASURY_FEE_RATE
+    );
+
+    assertEq(THE_MERGE_BLOCK + 1000, usdcTokenVault.campaignEndBlock());
+
+    vm.expectRevert(abi.encodeWithSignature("TokenVault_NotOwner()"));
```

Comment:
can you add more comment that since Frank is now the owner, the former owner cannot do anything